components/collective-page/hero/Hero.js

```diff
@@ -305,6 +324,16 @@ Hero.propTypes = {
     settings: PropTypes.shape({
       tos: PropTypes.string,
     }).isRequired,
+    memberOf: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.number.isRequired,
+        collective: PropTypes.shape({
+          id: PropTypes.number.isRequired,
+          name: PropTypes.string.isRequired,
+          // slug: PropTypes.string.isRequired,
```

Comment:
Let's remove rather than commenting

```suggestion
```

Reply:
This was a mistake, this shouldn't have been commented out